src/containerConfig.ts

```diff
@@ -33,7 +33,8 @@ async function registerExternalValues(): Promise<void> {
   const logger = jsLogger({ ...loggerConfig, prettyPrint: loggerConfig.prettyPrint, hooks: { logMethod } });
   container.register(SERVICES.LOGGER, { useValue: logger });
 
-  const metrics = new Metrics(SERVICE_NAME);
+  const attributes: Attributes = { name: SERVICE_NAME };
+  const metrics = new Metrics(attributes);
```

Comment:
In boilerplate, the Metrics class does not require an attribute
```suggestion
  const metrics = new Metrics();
```